Orange/widgets/data/owfile.py

```diff
@@ -21,6 +21,14 @@ def add_origin(examples, filename):
             var.attributes["origin"] = dir_name
 
 
+def is_url_valid(url):
+    try:
+        urllib.request.urlopen(url)
+        return True
+    except:
```

Comment:
The [documentation](https://docs.python.org/3.4/library/urllib.request.html#module-urllib.request) says it raises `URLError` on errors?